### ISSUE TITLE
[1.12.2]Lava Bucket Generator

### ISSUE THREAD

**Issue by newfengye** [gigaherz/Ender-Rift#52]
Lava Bucket Generator
After using the bucket does not stack.
I hope we can do some optimization. 
The power consumption is fast. 
I don't want to add too many technology modules for personal survival

**Comment by newfengye** [comment]
Sorry, my English is not very good.

**Comment by newfengye** [comment]
After the Lava Bucket burns, it returns to the empty Bucket. If it is not taken out, the empty Bucket will be lost after the fuel is used up

**Comment by newfengye** [comment]
Put a lava bucket in the generator and it will return to the empty bucket after combustion. Do not take out the bucket. When the fuel in the generator runs out, the empty bucket for the second combustion will disappear

**Comment by gigaherz** [comment]
Yes I understand that bug exists in the 1.12 version, I just have other priorities.
For what it's worth, the bug is fixed in newer versions, 1.12 is just very old.
